src/test/tests.json

```diff
@@ -1180,5 +1180,30 @@
             "code": 200,
             "status": "stream"
         }
+    }],
+    "dailymotion": [{
+        "name": "regular video",
+        "url": "https://www.dailymotion.com/video/x8t1eho",
+        "params": {},
+        "expected": {
+            "code": 200,
+            "status": "stream"
+        }
+    }, {
+        "name": "private video",
+        "url": "https://www.dailymotion.com/video/k41fZWpx2TaAORA2nok",
```

Comment:
please use another link from an account with an appropriate name :(

Reply:
Is it available now?

Reply:
not yet